src/components/Comment/Comment.tsx

```diff
@@ -551,7 +553,9 @@ export const Comment = ({
                                                             iconSide="left"
                                                         >
                                                             {/* We use this span to scope the styling */}
-                                                            <span>Reply</span>
+                                                            <span data-link-name="reply to comment">
```

Comment:
Not sure what frontend behaviour is, but should we differentiate this sign-in link from the above?

Reply:
That would be useful, but it's not differentiated on Frontend, so for now we probably need to keep it the same to avoid discrepancy in the date. When on full discussion-rendering on the platform we can probably separate.